client/src/Main.elm

```diff
@@ -1801,6 +1736,46 @@ level0Decoder =
         |> JDP.required "references" (Decode.list referenceDecoder)
 
 
+backboneDecoder : Decode.Decoder Backbone
+backboneDecoder =
+    let
+        decodeOverhang : String -> Decode.Decoder Bsa1Overhang
+        decodeOverhang str =
+            case stringToBsa1Overhang (String.trim str) of
+                Just oh ->
+                    Decode.succeed oh
+
+                _ ->
+                    Decode.fail "Not a valid overhang"
+
+        decodeBsmb1Overhang : String -> Decode.Decoder Bsmb1Overhang
+        decodeBsmb1Overhang bsmb1_str =
+            case stringToBsmb1Overhang (String.trim bsmb1_str) of
+                Just bsmb1 ->
+                    Decode.succeed bsmb1
+
+                _ ->
+                    Decode.fail "Not a valid Gateway site"
+    in
+    Decode.succeed Backbone
+        |> JDP.required "name" Decode.string
+        |> JDP.required "id" (Decode.int |> Decode.map String.fromInt)
+        |> JDP.optional "bsa1_overhang" (Decode.maybe (Decode.string |> Decode.andThen decodeOverhang)) Nothing
```

Comment:
Is the `Decode.maybe` really required here?

Reply:
I've changed it for the bsa1 overhang as well as for the bsmb1 overhang.